package.json

```
@@ -1,50 +1,60 @@
 {
   "name": "@nuxtjs/strapi",
-  "version": "0.1.9",
-  "description": "Strapi module for NuxtJS",
+  "version": "0.1.10",
+  "description": "Strapi module for Nuxt",
   "repository": "nuxt-community/strapi-module",
   "license": "MIT",
-  "author": "NuxtJS",
-  "contributors": [
-    {
-      "name": "Benjamin Canac <canacb1@gmail.com>"
-    }
-  ],
+  "sideEffects": false,
+  "main": "lib/module.js",
+  "typings": "types/index.d.ts",
```

```suggestion
  "typings": "./types/index.d.ts",
```
Don't know if it's just a problem I have, but without including the `./` at the start of the `"typings"` field my IDE doesn't pick up the typings.